plenum/common/constants.py

```diff
@@ -200,7 +200,7 @@ class LedgerState(IntEnum):
 
 VALID_LEDGER_IDS = (POOL_LEDGER_ID, DOMAIN_LEDGER_ID, CONFIG_LEDGER_ID)
 
-CURRENT_PROTOCOL_VERSION = PlenumProtocolVersion.STATE_PROOF_SUPPORT.value
+CURRENT_PROTOCOL_VERSION = PlenumProtocolVersion.TXN_REFACTORING_SUPPORT.value
```

Comment:
Please rename `TXN_REFACTORING_SUPPORT` to `TXN_FORMAT_1_0` support.

Reply:
ok